### PR TITLE
Issue 72 - Fix bug with some old objects trying to access w object at content view

### LINKED ISSUE #72: Exception when rendering content view
At our customer this exception rose in some of the embedder objects

```python
Traceback (innermost last):
  Module ZPublisher.Publish, line 138, in publish
  Module ZPublisher.mapply, line 77, in mapply
  Module ZPublisher.Publish, line 48, in call_object
  Module sc.embedder.content.embedder, line 433, in __call__
  Module Products.Five.browser.pagetemplatefile, line 125, in __call__
  Module Products.Five.browser.pagetemplatefile, line 59, in __call__
  Module zope.pagetemplate.pagetemplate, line 132, in pt_render
  Module five.pt.engine, line 93, in __call__
  Module z3c.pt.pagetemplate, line 149, in render
  Module chameleon.zpt.template, line 258, in render
  Module chameleon.template, line 191, in render
  Module chameleon.template, line 171, in render
  Module 7a533513996bf7a1dba0123176c30da5, line 622, in render
  Module 25d653446b3058f7434b90c0670fa02c, line 1383, in render_master
  Module 25d653446b3058f7434b90c0670fa02c, line 614, in render_content
  Module 7a533513996bf7a1dba0123176c30da5, line 610, in __fill_main
  Module 7a533513996bf7a1dba0123176c30da5, line 338, in render_main
  Module five.pt.expressions, line 154, in __call__
  Module five.pt.expressions, line 126, in traverse
  Module zope.traversing.adapters, line 136, in traversePathElement
   - __traceback_info__: (<Products.Five.metaclass.View object at 0x7f71a9ec1bd0>, 'w')
  Module zope.traversing.adapters, line 50, in traverse
   - __traceback_info__: (<Products.Five.metaclass.View object at 0x7f71a9ec1bd0>, 'w', ('text', 'render'))
LocationError: (<Products.Five.metaclass.View object at 0x7f71a9ec1bd0>, 'w')

 - Expression: "view/w/text/render" 
 - Filename:   ... er-1.2b2-py2.7.egg/sc/embedder/content/templates/view.pt
 - Location:   (line 29: col 44)
 - Source:     ... tal:content="structure view/w/text/render" />
                                          ^^^^^^^^^^^^^^^^^^
 - Arguments:  repeat: {...} (0)
               template: <ViewPageTemplateFile - at 0x7f71b7ec9890>
               views: <ViewMapper - at 0x7f71a9c0b490>
               modules: <instance - at 0x7f71cbd75e18>
               args: <tuple - at 0x7f71d301b050>
               here: <ImplicitAcquisitionWrapper justica-reabre-processo-que-investiga-morte-de-rubens-paiva at 0x7f71bd7f9460>
               user: <ImplicitAcquisitionWrapper - at 0x7f71a87ba7d0>
               nothing: <NoneType - at 0x8fe4d0>
               container: <ImplicitAcquisitionWrapper justica-reabre-processo-que-investiga-morte-de-rubens-paiva at 0x7f71bd7f9460>
               request: <instance - at 0x7f71a6848518>
               wrapped_repeat: <SafeMapping - at 0x7f71a6853470>
               traverse_subpath: <list - at 0x7f71b4673d40>
               default: <object - at 0x7f71d2f36560>
               loop: {...} (1)
               context: <ImplicitAcquisitionWrapper justica-reabre-processo-que-investiga-morte-de-rubens-paiva at 0x7f71bd7f9460>
               view: <View view at 0x7f71a9ec1bd0>
               translate: <function translate at 0x7f71a46a2230>
               root: <ImplicitAcquisitionWrapper Zope at 0x7f71a4f31280>
               options: {...} (0)
               target_language: pt_BR
```

I don't know the internals around this **w** object, but if change this expression to **context/text** the problem is gone.

### PR DESCRIPTION
**Pull request by rodfersou**
closes #72

### CODE REVIEW

**Comment by hvelarde** [comment]
@rodfersou there's an empty space in the template:

screen shot 2017-01-11 at 22 08 31

**Comment by rodfersou** [comment]
@hvelarde this happens because the alternate content is [aligned in the right with this css rule](https://github.com/simplesconsultoria/sc.embedder/blob/master/src/sc/embedder/skins/styles/embedder.css#L72-L74).

It was already like this.

File: src/sc/embedder/tests/keywords.robot [hunk 1]
```diff
@@ -56,9 +60,21 @@ Update
     Input Text  css=${description_selector}  ${description}
     Click Element  css=${image_input_selector}
     Select File  image.jpg
+    Wait For Condition  return tinyMCE.activeEditor != null
+    Execute Javascript
+    ...  var editors = tinyMCE.editors;
+    ...  if (editors[0] === undefined) {
```
Comment (rodfersou) [inline src/sc/embedder/tests/keywords.robot]: This code is used to access each editor at Plone 4.2
Unfortunately RF javascript expression don't allow to add a comment

Comment (hvelarde) [inline src/sc/embedder/tests/keywords.robot]: add a comment above the `Execute JavaScript` directive.

**Comment by hvelarde** [comment]
I'm going to refactor this.

**Pull request closed without merging**